tools/clang/lib/CodeGen/CGCall.cpp

```diff
@@ -498,7 +498,7 @@ CodeGenTypes::arrangeLLVMFunctionInfo(CanQualType resultType,
   // HLSL Change Ends
   assert(std::all_of(argTypes.begin(), argTypes.end(),
                      isCanonicalAsParam)); // HLSL Change - skip array when
-                                           // check isCanonicalAsParam
+  (void)isCanonicalAsParam;                // check isCanonicalAsParam
```

Comment:
nit: the comment belongs to the line above, not `(void)...`. Place `(void)...` on the next line.

Reply:
I sort of see the assert and the dummying out as of a piece. If it didn't require including a new header, I'd use DXASSERT_LOCALVAR, but it's also not very important either way.